code/iaas/accounts/src/main/resources/META-INF/cattle/accounts/spring-accounts-context.xml

```diff
@@ -46,6 +46,8 @@
                         <bean class="io.cattle.platform.spring.resource.SpringUrlListFactory" >
                             <property name="resources">
                                 <list>
+                                    <value>classpath:schema/token/token-auth.json</value>
+                                    <value>classpath*:schema/token/token-auth.json.d/**/*.json</value>
```

Comment:
There was an error in the order of auth handlers and admin needed to be able to read the token as a result. But i fixed it. This was not supposed to still be here.